documentation/modules/post/linux/gather/enum_commands.md

```diff
@@ -0,0 +1,3831 @@
+busybox gunzip
+busybox gzip
+busybox halt
+busybox head
+busybox hexdump
+busybox hostid
+busybox hostname
+busybox httpd
+busybox hwclock
+busybox i2cdetect
+busybox i2cdump
+busybox i2cget
+busybox i2cset
+busybox id
+busybox ifconfig
+busybox ifdown
+busybox ifup
+busybox init
+busybox insmod
+busybox ionice
+busybox ip
+busybox ipcalc
+busybox ipneigh
+busybox kill
+busybox killall
+busybox klogd
+busybox last
+busybox less
+busybox link
+busybox linux32
+busybox linux64
+busybox linuxrc
+busybox ln
+busybox loadfont
+busybox loadkmap
+busybox logger
+busybox login
+busybox logname
+busybox logread
+busybox losetup
+busybox ls
+busybox lsmod
+busybox lsscsi
+busybox lzcat
+busybox lzma
+busybox lzop
+busybox md5sum
+busybox mdev
+busybox microcom
+busybox mkdir
+busybox mkdosfs
+busybox mke2fs
+busybox mkfifo
+busybox mknod
+busybox mkpasswd
+busybox mkswap
+busybox mktemp
+busybox modinfo
+busybox modprobe
+busybox more
+busybox mount
+busybox mt
+busybox mv
+busybox nameif
+busybox nc
+busybox netstat
+busybox nl
+busybox nologin
+busybox nproc
+busybox nsenter
+busybox nslookup
+busybox nuke
+busybox od
+busybox openvt
+busybox partprobe
+busybox passwd
+busybox paste
+busybox patch
+busybox pidof
+busybox ping
+busybox ping6
+busybox pivot_root
+busybox poweroff
+busybox printf
+busybox ps
+busybox pwd
+busybox rdate
+busybox readlink
+busybox realpath
+busybox reboot
+busybox renice
+busybox reset
+busybox resume
+busybox rev
+busybox rm
+busybox rmdir
+busybox rmmod
+busybox route
+busybox rpm
+busybox rpm2cpio
+busybox run-init
+busybox run-parts
+busybox sed
+busybox seq
+busybox setkeycodes
+busybox setpriv
+busybox setsid
+busybox sh
+busybox sha1sum
+busybox sha256sum
+busybox sha512sum
+busybox shred
+busybox shuf
+busybox sleep
+busybox sort
+busybox ssl_client
+busybox start-stop-daemon
+busybox stat
+busybox static-sh
+busybox strings
+busybox stty
+busybox su
+busybox sulogin
+busybox svc
+busybox svok
+busybox swapoff
+busybox swapon
+busybox switch_root
+busybox sync
+busybox sysctl
+busybox syslogd
+busybox tac
+busybox tail
+busybox tar
+busybox taskset
+busybox tc
+busybox tee
+busybox telnet
+busybox telnetd
+busybox test
+busybox tftp
+busybox time
+busybox timeout
+busybox top
+busybox touch
+busybox tr
+busybox traceroute
+busybox traceroute6
+busybox true
+busybox truncate
+busybox tty
+busybox tunctl
+busybox ubirename
+busybox udhcpc
+busybox udhcpd
+busybox uevent
+busybox umount
+busybox uname
+busybox uncompress
+busybox unexpand
+busybox uniq
+busybox unix2dos
+busybox unlink
+busybox unlzma
+busybox unshare
+busybox unxz
+busybox unzip
+busybox uptime
+busybox usleep
+busybox uudecode
+busybox uuencode
+busybox vconfig
+busybox vi
+busybox w
+busybox watch
+busybox watchdog
+busybox wc
+busybox wget
+busybox which
+busybox who
+busybox whoami
+busybox xargs
+busybox xxd
+busybox xz
+busybox xzcat
+busybox yes
+busybox zcat
+[*] Post module execution completed
+msf6 post(linux/gather/enum_commands) > 
+```
```

Comment:
Perhaps some ellipses rather than listing out everything to a 4K like document.

Reply:
done